aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -678,14 +813,20 @@ module aptos_framework::account {
 
         // Set `OriginatingAddress[new_auth_key] = originating_address`.
         let new_auth_key = from_bcs::to_address(new_auth_key_vector);
-        table::add(address_map, new_auth_key, originating_addr);
+        table::add(
```

Comment:
Why do we need this line break here?

Reply:
Here, the line break is due to the length of the parameters inside the parentheses being too long. But this rule should just use for **Function Header**, not the function call. We will fix it

Reply:
fixed at https://github.com/movebit/movefmt/commit/a3e29fd6f6b715962de1b65192f48d84d70641cf